packages/preview-middleware/templates/flp/editor.html

```diff
@@ -10,7 +10,7 @@
     </head>
     <body>
         <noscript>You need to enable JavaScript to run this app.</noscript>
-        <div id="root"></div>
+        <div id="root" style="display: flex;flex-direction: column;"></div>
```

Comment:
This is questionable, would like the opinions of other reviewers on this. Maybe an extra div inside the root would do the trick?

Reply:
I don't think we need an extra div here, but we should have the styles in CSS files instead.